Multisig/Info.plist

```diff
@@ -95,6 +95,8 @@
 	<false/>
 	<key>IntercomAutoIntegratePushNotifications</key>
 	<string>NO</string>
+	<key>IntercomRegion</key>
```

Comment:
Can you help me understand why this change required ?

Reply:
whole point of this ticket. We are migrating Intercom from us to eu region

Reply:
https://developers.intercom.com/installing-intercom/docs/ios-data-hosting-region-configuration